sync/gh.py

```diff
@@ -88,7 +88,13 @@ def remove_labels(self, pr_id, *labels):
         pr_id = self._convert_pr_id(pr_id)
         issue = self.repo.get_issue(pr_id)
         for label in labels:
-            issue.remove_from_labels(label)
+            try:
+                issue.remove_from_labels(label)
+            except github.GithubException as e:
+                if e.data["message"] != "Label does not exist":
```

Comment:
I think we should just ignore the error if the label doesn't exist and log a warning in other cases (for bonus points report to new relic). I don't think we want to block the process on the failure to remove a label.